CONTRIBUTING.md

```diff
@@ -1,54 +1,57 @@
 # CONTRIBUTING
-
-Contributing to the official buyTogether Application
+Contributing to the official buyTogether application
 Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
-
 ---
-
-## Make changes locally
-
-- Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
-
-- Create a working branch 
-
-- Set up a development environment to begin working on the project all you need to do is 'vagrant up'
-
+## Prerequisites
+- Fork the repo to make your changes without affecting the original project until you're ready to merge.
+- Create a working branch.
+- Set up a development environment to begin working on the project, all you need to do is 'vagrant up'
 - Start with your changes!
-
 ---
-
-## Commit your update
-Commit the changes once you are happy with them. 
+## Commiting
+Please use the guide below inorder to follow our commit message convention:
+* [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
```

Comment:
This very commit does not follow the guide that you reference. e.g. the commit message title does not follow that format.

Also, I don't want the body to be optional. It should be mandatory.

Reply:
you're right, i didnt notice those details.